src/helm/benchmark/presentation/run_specs.conf

```diff
@@ -467,6 +467,33 @@ entries: [
   {description: "lsat_qa:model=all,task=assignment", priority: 3}
   {description: "lsat_qa:model=all,task=miscellaneous", priority: 3}
 
+  {description: "lextreme:subset=brazilian_court_decisions_judgment,model=all", priority: 3}
+  {description: "lextreme:subset=brazilian_court_decisions_unanimity,model=all", priority: 3}
+  {description: "lextreme:subset=german_argument_mining,model=all", priority: 3}
+  {description: "lextreme:subset=greek_legal_code_chapter,model=all", priority: 3}
+  {description: "lextreme:subset=greek_legal_code_subject,model=all", priority: 3}
+  {description: "lextreme:subset=greek_legal_code_volume,model=all", priority: 3}
+  {description: "lextreme:subset=swiss_judgment_prediction,model=all", priority: 3}
+  {description: "lextreme:subset=online_terms_of_service_unfairness_levels,model=all", priority: 3}
+  {description: "lextreme:subset=online_terms_of_service_clause_topics,model=all", priority: 3}
+  {description: "lextreme:subset=covid19_emergency_event,model=all", priority: 3}
+  {description: "lextreme:subset=multi_eurlex_level_1,model=all", priority: 3}
+  {description: "lextreme:subset=multi_eurlex_level_2,model=all", priority: 3}
+  {description: "lextreme:subset=multi_eurlex_level_3,model=all", priority: 3}
+  {description: "lextreme:subset=greek_legal_ner,model=all", priority: 3}
+  {description: "lextreme:subset=legalnero,model=all", priority: 3}
+  {description: "lextreme:subset=lener_br,model=all", priority: 3}
+  {description: "lextreme:subset=mapa_ner_coarse_grained,model=all", priority: 3}
+  {description: "lextreme:subset=mapa_ner_fine_grained,model=all", priority: 3}
```

Comment:
I ran into:

```
  File "/home/yifanmai/oss/helm/src/helm/benchmark/run_specs.py", line 1599, in get_lextreme_spec
    instructions=get_lextreme_instructions(subset),
  File "/home/yifanmai/oss/helm/src/helm/benchmark/scenarios/lextreme_scenario.py", line 184, in get_lextreme_instructions
    return INSTRUCTIONS[subset]
KeyError: 'mapa_ner_coarse_grained'
```

I think you need to change "mapa_ner_coarse_grained" to "mapa_coarse" and "mapa_ner_fine_grained" to "mapa_fine".

Reply:
you are right, thanks. Fixed it.